website/project/views/node.py

```diff
@@ -1008,6 +1008,66 @@ def get_children(auth, node, **kwargs):
     return _render_nodes(nodes, auth)
 
 
+def node_child_tree(user, node_ids):
```

Comment:
I tried Brian's approach, but it provides a list of nodes, not a tree.  and I need it for the treebeard instance, at least until V2 works with treebeard.

There is similar code for the wiki and logs, but I would need to modify it pretty heavily to make it work.  I worry it would slow down the existing code.